public/app/plugins/datasource/loki/components/LokiQueryEditor.tsx

```diff
@@ -104,9 +104,10 @@ export const LokiQueryEditor = React.memo<LokiQueryEditorProps>((props) => {
       query.queryType,
       previousQueryType
     );
-    if (update) {
+    // Time range check duplicated to satisfy typescript. Already checked in shouldUpdateStats.
+    if (update && timeRange) {
```

Comment:
```suggestion
    if (update && timeRange) {
```
Minor, but I don't think we need to make a comment on every "if" that TypeScript persuades us to add. I personally find the whole concept of "make TS happy" negative.

Reply:
I agree.